src/include/splash/basetypes/ColTypeDim.hpp

```diff
@@ -88,7 +88,13 @@ namespace splash
             else
                 return NULL;
         }
-    };
+
+        static size_t getSize_()
```

Comment:
Ups. I meant to. The compile error was about calling an object method from a static function so I need a static function but also an override for the pure virtual function that does the same